src/SoundList.js

```diff
@@ -32,13 +32,23 @@ const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
                   {track.username}
                 </span>
               </div>
-              <div className="hidden sm:flex flex-row justify-end items-center opacity-50 group-hover:opacity-100 sm:hover:opacity-0 space-x-12">
-                <IonIcon className="pr-4" icon={timeSharp} size="large" />
-                {timeConvert(track.duration)}
-                <IonIcon className="pr-4" icon={downloadSharp} size="large" />
-                {track.num_downloads}
-                <IonIcon className="pr-4" icon={playCircleSharp} size="large" />
-                {track.num_ratings}
+              <div className="w-6/12 md:w-4/12 md:space-2 hidden sm:flex flex-row justify-end items-center opacity-50 group-hover:opacity-100 sm:hover:opacity-0">
+                <div className="flex justify-end w-4/12 items-center">
+                  <IonIcon className="p-2" icon={timeSharp} size="large" />
+                  {timeConvert(track.duration)}
+                </div>
+                <div className="flex justify-end w-4/12 items-center">
```

Comment:
we should revert the change to justify-end and i think we can merge this

Reply:
what do you mean by revert? It is currently `justify-end
`

Reply:
do you mean revert it back to `justify-start`?

Reply:
yes